src/ray/core_worker/transport/task_receiver.cc

```diff
@@ -169,19 +169,17 @@ void TaskReceiver::HandleTask(const rpc::PushTaskRequest &request,
       }
 
       if (task_spec.IsActorCreationTask()) {
-        /// The default max concurrency for creating PoolManager should
-        /// be 0 if this is an asyncio actor.
-        const int default_max_concurrency =
-            task_spec.IsAsyncioActor() ? 0 : task_spec.MaxActorConcurrency();
-        pool_manager_ = std::make_shared<ConcurrencyGroupManager<BoundedExecutor>>(
-            task_spec.ConcurrencyGroups(),
-            default_max_concurrency,
-            initialize_thread_callback_);
         if (task_spec.IsAsyncioActor()) {
           fiber_state_manager_ = std::make_shared<ConcurrencyGroupManager<FiberState>>(
               task_spec.ConcurrencyGroups(),
               fiber_max_concurrency_,
               initialize_thread_callback_);
+        } else {
```

Comment:
drop a comment on why we don't need the concurrency group manager for asyncio

Reply:
https://github.com/ray-project/ray/pull/51069/commits/ceace22d7fdf01b2c35b5457a313bca1d9272353 added